### PR TITLE
[serve] Fix `app_builder` doc code test

### PR DESCRIPTION
**Pull request by edoakes**
## Why are these changes needed?

Missing import.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/serve/doc_code/app_builder.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ def app_builder(args: Dict[str, str]) -> Application:
 
 # __end_untyped_builder__
 
+import requests
```
Comment (sihanwang41) [inline doc/source/serve/doc_code/app_builder.py:26]: how does it pass the ci?

**Review by sihanwang41 — APPROVED** [review]

File: doc/source/serve/doc_code/app_builder.py (hunk 1 above)
Comment (edoakes) [reply doc/source/serve/doc_code/app_builder.py:26]: it doesn't :)

**Review by angelinalg — APPROVED** [review]
LGTM
But I can't review the code.

**Review by zcin — APPROVED** [review]

**Comment by ArturNiederfahrenhorst** [comment]
@sihanwang41 @edoakes I think this needs to go into 2.5.0 release branch, please confirm.
Relevant build: https://buildkite.com/ray-project/oss-ci-build-branch/builds/3954#01882765-5252-48f6-9bad-c4ea2e9b5f1e

CC @zhe-thoughts

**Comment by edoakes** [comment]
Yep I can make a cherry-pick PR

**Pull request merged**